### ISSUE TITLE
Implementierung langwelliges Strahlungsmodell

### ISSUE THREAD

**Issue by Arombolosh** [ghorwin/SIM-VICUS#513]
Der langwellige Strahlungsmodell ist derzeit nicht implementiert.

Implementierung:

* Langwellige Himmelsgegenstrahlung kommt aus den Klimadatensatz
* Erdreichtemperatur wird als Umgebungstemperatur angenommen.
* Emissionskoeffizient ist im NANDRAD Modell erhalten
* Gesetz von Boltzmann anwenden und implementieren

Zusätzlich auch die Ausgaben über die NANDRAD Datei bereitsstellen

**Comment by ghorwin** [comment]
Das _langweilige_ Strahlungsmodell würde ich dann mal einbauen - hab das ja zuletzt bei DELPHIN erfolgreich reviewed und doklumentiert.

Soll es auch noch eine richtungsabhängige Erdreichtemperatur geben?
Das Thema "Klimadatensatz" bearbeiten und ggfs fehlenden Himmelsgegenstrahlung irgendwie aus anderen Messwerten/Modellen ableiten, wäre dann ein paralleles Thema (siehe die langjährigen Bemühung von Peter Häupl).

@Stephan: wird das auch ein neuer SimQuality-Testfall? Ausrichtungen der Wand: 0, 45,90,180°?

**Comment by hirseboy** [comment]
Ja, fände ich gut. Aber dann außerhalb des Projektes. Da schreibe ich gerade den Abschlussbericht.

LG

**Comment by ghorwin** [comment]
Natürlich, hatte das eher als "sowieso Prüfung" für die Modellimplementierung gedacht.

**Comment by ghorwin** [comment]
Damit hatte sich Anne vor einigen Jahren intensiv beschäftigt.
